core/domain/rights_manager.py

```
@@ -164,7 +164,8 @@ def _save_activity_rights(
             committer_id, activity_rights, commit_message, commit_cmds
         )
         return
-    elif activity_type == constants.ACTIVITY_TYPE_COLLECTION:
+    else:
```

Ruff (0.15.11)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)